presentation.tex

```diff
@@ -9,8 +9,28 @@
 \date[ISBT 2018]{27th International Symposium of Beamer Themes}
 \author[DB]{Dries Benoit --- \texttt{dries.benoit@ugent.be}}
 
+% Macro aimed at loading themes in different directories
+\makeatletter
+  \def\beamer@calltheme#1#2#3{%
+    \def\beamer@themelist{#2}
+    \@for\beamer@themename:=\beamer@themelist\do
+    {\usepackage[{#1}]{\beamer@themelocation/#3\beamer@themename}}}
+
+  \def\usefolder#1{
+    \def\beamer@themelocation{#1}
+  }
+  \def\beamer@themelocation{}
+
 % Load the UGent theme
+\usefolder{theme}
 \usetheme[language=en,faculty=eb,usecolors]{ugent}
```

Comment:
```suggestion
\usetheme[language=en,faculty=we,usecolors]{ugent}
```
Maybe it would be easier to start out on "faculty of science", since the template is aimed at our group?